tests/services/test_sarses_generator.py

```diff
@@ -0,0 +1,225 @@
+# pylint: disable-all
+
+import logging
+import pytest
+from app.services.sarses_generator import generate_sarses
+from tests.factories.marketplace_factories import UserFactory
+from tests.factories.recommendation_factory import RecommendationFactory
+from tests.factories.user_action_factory import UserActionFactory
+from app.db.mongo_models import Sars
+from tests.fixtures.utils import mongo
+from app.reward_mapping import ua_to_reward_id
+from app.services.sarses_generator import (
+    RECOMMENDATION_PAGES_IDS,
+    _tree_collapse,
+    _get_clicked_services_and_reward,
+    _find_root_uas_before,
+)
+from app.db.mongo_models import UserAction
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
Is this used anywhere?

Generally test files do not use package scoped loggers...